ds_judgements_public_ui/templates/includes/card_homepage.html

```diff
@@ -10,6 +10,7 @@
     <h3>
       <a href="{% url link_route %}">{{ title }}</a>
     </h3>
-    <p>{{ body }}</p>
+
   </div>
```

Comment:
There is a breakpoint where these cards look quite off:
<img width="768" alt="image" src="https://github.com/user-attachments/assets/5c6ccc9f-f364-45b4-a95e-d80029789c75">

Reply:
@jlhdxw thanks updated